lang/1031.json

```diff
@@ -144,6 +144,14 @@
         "start_with_windows": "PicoTorrent mit Windows starten",
         "start_position": "Startposition",
         "hidden": "System Tray",
-        "minimized": "Minimiert"
+        "minimized": "Minimiert",
+        "_torrent_s": "%d Torrent(s)",
+        "queuing": "in Warteschlange",
+        "up": "Hoch",
+        "down": "Runter",
+        "top": "Oben",
+        "bottom": "Unten",
+        "remove_tracker_s": "Tracker entfernen",
```

Comment:
I actually don't know the rules of how German forms its plurals, but there are definately words that don't change, and even though "tracker" isn't a German word, it sounds correct to my ears to use say "1 Tracker, 2 Tracker" in German, but as i said i can't tell you why ;D